wrapanapi/systems/virtualcenter.py

```diff
@@ -1096,3 +1279,22 @@ def get_network(self, network_name):
         if not network:
             raise NotFoundError
         return network
+
+    def get_datastore(self, datastore_name):
+        """Fetch the Datastore object(for a datastor) or StoragePod object (for a datastore
+        cluster) from a provided name
+
+        Args:
+            datastore_name: The name of the datastore or datastore cluster
+        Returns: A object of vim.Datastore or vim.StoragePod
+        """
+
+        if datastore_name in self.list_datastore():
```

Comment:
is there a chance that some datastore cluster and some datastore have the same name

Reply:
I faced the same issue and asked KK, the answer was no. But I can put in a check that it is also not in list_datastore_cluster()